### PR TITLE
DBR-313 - Semantic Release

### PR DESCRIPTION
**Pull request by andresmoschini**
Hi team,

This is an end-user application, so, maybe semantic versioning is not the best option, but maybe it is useful because we need number versions, and it is automatic and gives us some kind of information.

Following (in part) [this article](https://medium.com/@u_glow/making-sense-of-semantic-versioning-for-end-user-software-applications-a3049d97478b) we can apply these rules:

* `MAJOR` version  (`BREAKING CHANGES`): when we make changes incompatible with current Doppler Backend, DataHub ApiSec, or other backend APIs. Also when we remove or change drastically some end-user features. And also when our changes affect the installation-requirements or infrastructure.
* `MINOR` version (`feat`): when we add functionality in a backwards-compatible manner (e.g. passing additional data to an already-provisioned API or adding any end-user functionality that does not affect the installation-requirements)
* `PATCH` version (`fix`): when we make backwards-compatible bug fixes (e.g. fixing any end-user bug that does not affect the installation requirements).

Regarding performance or security improvements, in my opinion, we need to consider them as _fixes_ or even _features_, in order to increment the version number.

What do you think?

Please review it well, because I cannot test it before merging.

### CODE REVIEW

File: build-n-release-w-docker.sh [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+
+echo Publishing to Akamai...
+echo pkgName: $pkgName
+echo pkgVersion: $pkgVersion
+echo cdnBaseUrl: $cdnBaseUrl
+
+# Force pull the latest image version due to the cache not always is pruned immediately after an update is uploaded to docker hub
+docker pull dopplerrelay/doppler-relay-akamai-publish
+
+docker run --rm \
```
Comment (cbernat) [inline build-n-release-w-docker.sh]: This is production build right?

Comment (andresmoschini) [reply build-n-release-w-docker.sh]: Mmm, it depends on what is "production" for you.

It copies the result files of production build (the result of `...&& yarn semantic-release`) to our CDN.

Does it answer your question?

**Comment by andresmoschini** [comment]
test this please

File: build-n-release-w-docker.sh (hunk 1 above)
Comment (cbernat) [reply build-n-release-w-docker.sh]: So it is production then 👍

**Pull request merged**